@here/harp-webtile-datasource/lib/WebTileDataSource.ts

```diff
@@ -26,6 +26,12 @@ export interface WebTileRenderingOptions {
      * @default false
      */
     transparent?: boolean;
+
+    /**
+     * RenderOrder for order in which to render WebTileDataSouurces
+     * @default 0
+     */
+    renderOrder?: number;
```

Comment:
Just wondering if we go in the right direction here. We add more and more styling options to the API but maybe we should just make this datasource styleable (we also have a ticket to make landmarks styleable).

Reply:
Very good point, yes, I think it would make sense to actually make it stylable. As this will need some more evaluation and gathering of requirements etc, I ll create a ticket for it, to not block the current work.